tests/Application/Actions/Donations/CancelAllTest.php

```diff
@@ -183,6 +186,7 @@ private function setDoublesInContainer(
         ObjectProphecy $stripeProphecy = null,
         ObjectProphecy $allocatorProphecy = null,
     ): void {
+        $container->set(CampaignFundingRepository::class, $this->prophesize(CampaignFundingRepository::class)->reveal());
```

Comment:
or
```suggestion
$container->set(Allocator::class, $this->prophesize(Allocator::class)->reveal());
```

which would be more similar to what we have now. DonationRepository is splitting into DonationRepository and Allocator so we would now fake both of them instead of just the former.